azure/notebooks/kubernetes/Dask_Kube_cuML_Exploration_Full.ipynb

```diff
@@ -0,0 +1,1362 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def construct_worker_pool(client, n_workers, auto_scale=False, timeout=300):
</pre><p>Where is this function being used?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cloud-ml-examples/pull/118/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Same question</p>

Reply:
<p>Not using actually. Removed. </p>